Bikeini/screens/AddBike.js

```diff
@@ -348,6 +348,10 @@ class AddBike extends React.Component {
       Alert.alert('Please set a location');
       return;
     }
+    if (!bikeData.title || !bikeData.brand || !bikeData.description) {
+      Alert.alert('Please make sure your ad contains a title, brand-information and description');
+      return;
```

Comment:
The stars (*) that are used for showing which fields are mandatory are not completely correct!
Model and Brand seems to been swapped 😄